1-exercises/G-array-methods/exercise.js

```diff
@@ -4,7 +4,7 @@
 */
 
 let numbers = [3, 2, 1];
-let sortedNumbers; // complete this statement
+let sortedNumbers= numbers.sort(); // complete this statement
```

Comment:
this will work in this case, however the `.sort()` method in javascript is silly in that it treats all values as strings all the time, meaning that `[1,3,11,23].sort() = [1,11,23,3]` - so make sure to always pass a comparison function https://www.w3schools.com/js/js_array_sort.asp